Karma/Karma/CSVProcessing.cs

```diff
@@ -9,10 +10,54 @@ namespace Karma
 {
     public class CSVProcessing
     {
-        public void AppendToCSV(string text)
+        public static Item[] Items;
```

Comment:
I do agree that the Methods in this class should be static, however, I don't think Items should be in this class, since this class is supposed to act as a Utilities class for file reading. It's fine for now I guess, but in the future when we will be working with databases, it should be different